app/Exceptions/Handler.php

```diff
@@ -83,7 +81,7 @@ public function register(): void
     protected function unauthenticated($request, AuthenticationException $exception)
     {
         return $this->shouldReturnJson($request, $exception)
-            ? response()->json(['message' => $exception->getMessage()], 401)
-            : redirect()->guest($exception->redirectTo() ?? url('/'));
+          ? response()->json(['message' => $exception->getMessage()], 401)
+          : redirect()->guest($exception->redirectTo() ?? url('/'));
```

Comment:
Though I am fine with the changes to `$levels` above, this reduces tabbing down to 2 spaces for these lines. This is against our formatting rules and I am not sure why CI is not fixing this.

Regardless I have seen this in a few of your PRs. Please check your IDE and change it so tabbing is 4 spaces, to 2. It may also be good to setup your IDE to use our linting rules. There is a PHPCSFixer plugin recommended in the Loop document for VSCode.

Reply:
I have setup PHPCSFixer plugin in my IDE still even i am not sure why this is happening